airflow/www/extensions/init_views.py

```diff
@@ -58,7 +58,6 @@ def init_appbuilder_views(app):
     appbuilder.add_view(views.PoolModelView, "Pools", category="Admin")
     appbuilder.add_view(views.VariableModelView, "Variables", category="Admin")
     appbuilder.add_view(views.XComModelView, "XComs", category="Admin")
-    appbuilder.add_view(views.VersionView, 'Version', category='About', category_icon='fa-th')
```

Comment:
Would be good to see the Screenshot of the footer where we can see the Airflow Version information

Reply:
Ah, yes! Added this to the PR description too:
![image](https://user-images.githubusercontent.com/3267/94587939-44836380-0251-11eb-8514-f23dc2aa7d62.png)